jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/DefaultAwsConnectionManagerTest.kt

```diff
@@ -72,7 +69,7 @@ class DefaultAwsConnectionManagerTest {
 
     @Test
     fun `Starts with no active credentials`() {
-        assertThat(manager.isValidConnectionSettings()).isFalse()
+        assertThat(manager.isValidConnectionSettings()).isFalse
```

Comment:
intentional?

Reply:
It was, since the IDE was marking all the lines.

I could understand wanting to undo these to keep the commit cleaner though. Let me know